tests/compat_contextlib.py

```diff
@@ -0,0 +1,20 @@
+from contextlib import AbstractContextManager
+
+
+class nullcontext(AbstractContextManager):
```

Comment:
Might be worth using contextlib2.nullcontext?

Reply:
Ah yes, could do. It's just for the tests so I did a very quick copy-paste from cpython, but can use contextlib2 instead. Will follow up tomorrow.

Reply:
Updated here: https://github.com/jazzband/pathlib2/commit/2c8f4b1b842efce9fda4ca67c2b1b0633bedc052